src/test/java/org/broadinstitute/hellbender/engine/GenomicsDBIntegrationTest.java

```diff
@@ -33,6 +33,15 @@ public void testGenomicsDBInClassPath(){
         Assert.assertNotNull(GenomicsDBLibLoader.class.getResource(path), "Could not find the genomicsdb binary at " + path);
     }
 
+    @Test
```

Comment:
Test to check if both - Linux and MacOS shared libraries - are packaged in the genomicsdb jar.